utils/envvars/parser.go

```diff
@@ -141,6 +148,44 @@ func parseStringDuration(value string) (time.Duration, error) {
 	return d, nil
 }
 
+// GetSAASHost returns SAAS host env variable value if it's valid.
+// Otherwise returns defaultSAASHost.
+func GetSAASHost(env string) (string, error) {
+	if v, ok := os.LookupEnv(envSAASHost); ok {
+		h, p, err := net.SplitHostPort(v)
+		if err != nil {
+			return v, err
```

Comment:
```suggestion
			return nil, err
```

Reply:
As I see then I should change the return type to `*string` is that correct?

Reply:
Totally idiomatic way would be `return "", err` so the reader does not have to think too much about if v is empty or not there (cases when both returned value and error are set are rare, but exist: https://golang.org/pkg/io/#Reader)

But that's not that important there

Reply:
What _is_ important that that we duplicated some logic logic like calling SplitHostPort, etc. See https://github.com/percona/pmm-managed/pull/466/commits/1f09b8baa2a655350d335c141314e0e25f8d2ff8

Reply:
thank you